source/_components/insteon_local.markdown

```diff
@@ -2,15 +2,15 @@
 layout: page
 title: "Insteon (local)"
 description: "Instructions how to setup the Insteon Hub locally within Home Assistant."
-date:  2016-12-18 08:00
+date:  2017-12-11 08:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: insteon.png
 ha_category: Hub
 ha_iot_class: "Local Polling"
-ha_version: 0.36
+ha_version: 0.59.2
```

Comment:
This metadata should just stay as it was. It reflects the first version of hass that included it.